### PR TITLE
Minor updates for new names, adds phones postprocess

### LINKED ISSUE #360: wikipron/data/src reorganization
[`wikipron/data/src`](https://github.com/kylebgorman/wikipron/tree/master/data/src) is host to a lot of different files that do a lot of different things in different places. The unorganized state of `wikipron/data/src` may be off-putting to new/prospective contributors who don’t want to wade into this large collection of files to try and get their bearings.

I have a few suggestions for things we can do to reorganize `wikipron/data/src`, though I’d also like suggestions for how (or even if) `wikipron/data/src` should be reorganized from those who contributed some of these files.

A few things come to mind immediately:
- We wrote `postprocess` but for whatever reason only call `split.py` from within it. We should call `generate_tsv_summary.py` from `postprocess` as well and maybe put both `split.py` and `generate_tsv_summary.py` in an `src/postprocess_helpers` subdirectory.
- All of our `.json` files should go in a subdirectory (or at least all of them that are not `languages.json`).
- The “Big Scrape” Scripts README is out of date, we no longer retry languages 10 times.
- The scripts which have to do with the phones stuff should be in there own subdirectory? There are quite a few of them now.
- `covering_grammar.py` seems a bit homeless. I don’t think we’ve used it.

I’m not sure if putting things in subdirectories of `src` is a great solution, so any alternatives would be welcome.

### PR DESCRIPTION
**Pull request by lfashby**
- [x] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

This just contains some small changes now that we are using 'broad' and 'narrow'. I don't think we really need to change anything else. Apparently our phones table was broken, so I changed the links.

I added a postprocess script to the phones side just for the sake of consistency.

Closes #360.

### CODE REVIEW

File: data/scrape/tsv_summary.tsv [hunk 1]
```diff
@@ -11,10 +11,10 @@ ale_latn_broad.tsv	ale	Aleut	Aleut	Latin		False	Broad	True	104
 grc_grek_broad.tsv	grc	Ancient Greek (to 1453)	Ancient Greek	Greek		False	Broad	True	90222
 ara_arab_broad.tsv	ara	Arabic	Arabic	Arabic		False	Broad	False	7279
 arc_hebr_broad.tsv	arc	Imperial Aramaic (700-300 BCE); Official Aramaic (700-300 BCE)	Aramaic	Hebrew		False	Broad	False	1156
+arm_armn_e_narrow.tsv	arm	Armenian	Armenian	Armenian	Eastern Armenian, standard	False	Narrow	True	14182
 arm_armn_w_narrow.tsv	arm	Armenian	Armenian	Armenian	Western Armenian, standard	False	Narrow	True	14065
 arm_armn_w_narrow_filtered.tsv	arm	Armenian	Armenian	Armenian	Western Armenian, standard	True	Narrow	True	14040
 arm_armn_e_narrow_filtered.tsv	arm	Armenian	Armenian	Armenian	Eastern Armenian, standard	True	Narrow	True	14177
```
Comment (ajmalanoski) [inline data/scrape/tsv_summary.tsv:17]: Very minor thing, but maybe we just do finer sorting than just Wiktionary name and transcription level, since it gives  us a random sorting of the four Armenian files (all of which have the same Wiktionary name and transcription level) and of other similar cases.

**Comment by ajmalanoski** [comment]
I had one minor suggestion, but otherwise LGTM. Thanks for catching the issue with the links in the HOWTO.

File: data/phones/lib/generate_phones_summary.py [hunk 2]
```diff
@@ -68,34 +52,28 @@ def main() -> None:
...
         row = [
             iso639_code,
             languages[iso639_code]["iso639_name"],
             wiki_name,
             transcription_level,
             num_of_entries,
         ]
-        languages_summary_list.append([file_path] + row)
-        readme_list.append([f"[phone]({file_path})"] + row)
+        phones_summary_list.append([file_path] + row)
```
Comment (kylebgorman) [inline data/phones/lib/generate_phones_summary.py:66]: drive-by, but `phones_summaries` is probably a better variable name than `phones_summary_list`.

File: data/phones/postprocess [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Runs summary generation script.
+
+set -eou pipefail
+
+main() {
```
Comment (kylebgorman) [inline data/phones/postprocess:6]: If this is just one line I think you don't have to use a `main`. I just use it when there are multiple steps.

File: data/scrape/tsv_summary.tsv (hunk 1 above)
Comment (kylebgorman) [reply data/scrape/tsv_summary.tsv:17]: +1

Comment (lfashby) [reply data/scrape/tsv_summary.tsv:16]: I changed it to sort on wiki name, then whether or not they're filtered (just so that they are grouped, not really necessary but maybe looks nice), and on the path -- which has to be unique and contains transcription level anyway.

File: data/scrape/lib/generate_tsv_summary.py [hunk 4]
```diff
@@ -15,8 +15,8 @@
 )
 
 
-def _wiki_name_and_transcription_level(ele: List[str]) -> str:
-    return ele[3] + ele[7]
+def wiki_name_filtered_and_path(ele: List[str]) -> str:
```
Comment (kylebgorman) [inline data/scrape/lib/generate_tsv_summary.py:18]: Would it not suffice to just sort by the path? That'd make this function unnecessary, could do `operator.itemgetter(0)` instead, which is way faster too.

Comment (lfashby) [reply data/scrape/lib/generate_tsv_summary.py:18]: Perhaps, but then we'd be sorting by ISO code and get things like Irish ending up next to German.

**Comment by kylebgorman** [comment]
FWIW that doesn't bother me.

**Comment by ajmalanoski** [comment]
I think we can close it. The original goal of the issue (i.e.,`wikipron/data/src` reorganization) has been met, IMO — `data/src` has been thoroughly reorganized (and removed).

There is still one small outstanding issue related to the project of reorganization, namely the suggested renaming/reorganization of `data/covering_grammar` (@Othergreengrasses), but I don't think it directly relates to the discussion in #360

**Comment by kylebgorman** [comment]
Okay. Add it to the PR description then, Lucas, if you could.

K

**Comment by lfashby** [comment]
I've hopefully sorted out our sorting problem. I can go through tomorrow and add some content to our temporary READMEs. I'll add the relevant closing statement then.

File: data/phones/lib/generate_phones_summary.py [hunk 5]
```diff
@@ -68,34 +49,28 @@ def main() -> None:
...
             languages[iso639_code]["iso639_name"],
             wiki_name,
             transcription_level,
             num_of_entries,
         ]
-        languages_summary_list.append([file_path] + row)
-        readme_list.append([f"[phone]({file_path})"] + row)
+        phones_summaries.append([file_path] + row)
+        readme_list.append([f"[phone](phones/{file_path})"] + row)
     # Sorts by Wiktionary language name, with broad entries before narrow
```
Comment (kylebgorman) [inline data/phones/lib/generate_phones_summary.py:65]: Is this comment still accurate?

File: data/phones/lib/generate_phones_summary.py [hunk 6]
```diff
@@ -5,7 +5,8 @@
 import logging
 import os
 
-from typing import Any, Dict, List
+from operator import itemgetter
```
Comment (kylebgorman) [inline data/phones/lib/generate_phones_summary.py:8]: Sort of a departure from the normal style here.

**Review by kylebgorman — APPROVED** [review]
LGTM, a few comments.

File: data/phones/lib/generate_phones_summary.py (hunk 6 above)
Comment (lfashby) [reply data/phones/lib/generate_phones_summary.py]: Good point!

**Comment by lfashby** [comment]
I didn't update the README in the covering grammar section as that directory is subject to change shortly.

**Review by kylebgorman — APPROVED** [review]
LGTM.

**Comment by kylebgorman** [comment]
Did you intend for this to be to a non-master branch? I just noticed that it is. Can I move it to master?

**Comment by lfashby** [comment]
Yeah you can move it to master, it was just building on top of Aidan's changes but I think we're done renaming stuff now.
